src/javascript/app/Stores/Modules/Trading/Helpers/symbol.js

```diff
@@ -1,4 +1,26 @@
+import { LocalStore } from '_common/storage';
+
 export const pickDefaultSymbol = (active_symbols = []) => {
     if (!active_symbols.length) return '';
+    return getFavoriteOpenSymbol(active_symbols) || getFirstOpenSymbol(active_symbols);
+};
+
+const getFavoriteOpenSymbol = (active_symbols) => {
+    const chart_favorites = LocalStore.get('cq-favorites');
+    if (!chart_favorites) return undefined;
```

Comment:
One minor thing: can we add a space after this? I think that would make it easier to read.